dts/msm8974/msm8974-fairphone-fp2.dts

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "skeleton.dtsi"
+
+/ {
+	model = "Fairphone 2";
+	compatible = "fairphone,fp2", "qcom,msm8974", "lk2nd,device";
+	qcom,msm-id = <208 0x10000>, <209 0x10000>, <211 0x10000>,
+		      <212 0x10000>, <214 0x10000>, <215 0x10000>,
+		      <217 0x10000>, <218 0x10000>;
```

Comment:
Do we really need all of them? :O

seems like many of them mean the same... https://github.com/msm8916-mainline/android_kernel_qcom_msm8916/blob/b20608408caff817ec874f325127b07609fbaeb8/drivers/soc/qcom/socinfo.c#L333-L355

Reply:
Blame Qualcomm :P https://github.com/LineageOS/android_kernel_fairphone_msm8974/blob/lineage-18.1/arch/arm/boot/dts/msm8974pro-ab-pm8941.dtsi#L16-L23

But I know that only MSM8974-AA and MSM8974-AB are used in the FP2, so 217 & 218 should be enough https://github.com/torvalds/linux/blob/master/drivers/soc/qcom/socinfo.c#L214-L215

Reply:
My phone has this btw: `[160] Board: platform: 217, foundry: 0x1, platform_version: 0x10001, platform_hw: 0x8, platform_subtype: 0x0, target: 0x10008, baseband: 0x0, platform_hlos_subtype: 0x0`